tests/set1.py

```diff
@@ -0,0 +1,9 @@
+def test_Set():
+    a: i32
+    b: i32
+
+    a = {1, 2, 3}
+    a = {2, 3, 4, 5, 5}
+    a = {"a", "b", "c"}
+    # a = {-1, -2, "c"} -> semantic error for now
+    b = a[0]
```

Comment:
Should this test be valid?
In python:
```py
>>> s={1,2,5,3,4}
>>> s[2]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'set' object is not subscriptable
```

Reply:
Thanks for pointing. We should remove this.

Reply:
Removed in https://github.com/lcompilers/lpython/pull/98